pkg/commands/loaders/stash.go

```diff
@@ -65,8 +65,8 @@ outer:
 }
 
 func (self *StashLoader) getUnfilteredStashEntries() []*models.StashEntry {
-	rawString, _ := self.cmd.New("git stash list --pretty='%gs'").DontLog().RunWithOutput()
-	return slices.MapWithIndex(utils.SplitLines(rawString), func(line string, index int) *models.StashEntry {
+	rawString, _ := self.cmd.New("git stash list -z --pretty='%gs'").DontLog().RunWithOutput()
```

Comment:
do we know which version of git added the -z arg to git stash? Cos if it's recent this'll break for some users

Reply:
Prior to v2.22.0, `git stash list` was implemented as a shell script using `git-log` (equivalent to `git log --format="%gd: %gs" -g -z "$@" refs/stash --`).
So at least as of v2.0.0, all options of `git-log` including `-z` are available.
https://github.com/git/git/blob/e156455ea49124c140a67623f22a393db62d5d98/git-stash.sh#L299

Reply:
ah nice!